docs/source/use/using.rst

```diff
@@ -0,0 +1,172 @@
+=================
+Use Jupyter tools
```

Comment:
What do you think about calling this section "About Jupyter tools" or just "Jupyter tools"?
"Use Jupyter tools" sounds a bit imperative in my head even without the exclamation mark.
I'm not a native speaker so that might be the cause :sweat_smile:

Reply:
hmmm - maybe "Using Jupyter tools"? I was trying to differentiate it as a section for "users" of Jupyter, as opposed to generally covering the jupyter community etc. But if you think it's clear-enough as "Jupyter Tools" I like that too. WDYT?

Reply:
*Using Jupyter Tools* is perfect.